util/Cargo.toml

```diff
@@ -3,7 +3,7 @@ name = "mycelium-util"
 version = "0.1.0"
 authors = ["Eliza Weisman <eliza@elizas.website>"]
 edition = "2021"
-rust-version = "1.61.0"
+rust-version = "1.76.0"
```

Comment:
and again, i don't think this is *really* necessary...although it's also fine since this isn't a published crate.